src/main/java/com/hotelbooking/numbers/FiveStarNumber.java

```diff
@@ -0,0 +1,25 @@
+package com.hotelbooking.numbers;
+
+import java.util.ArrayList;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class FiveStarNumber {
+    public  ArrayList<String>  getRoomParams() {
```

Comment:
Why do you keep all room characteristics in a List<String> instead of creating a separate Room model with needed fields? All of the Number classes actually represent the same object. You may significantly reduce code duplication if you just leave 1 class and create different instances of this class that will have different characteristics